modules/touch/include/touchinteraction.h

```diff
@@ -44,14 +44,31 @@ namespace openspace {
 class Camera;
 class SceneGraphNode;
 
+//Class used for keeping track of the recent average frame time
+class FrameTimeAverage
+{
+public:
+    //Update the circular buffer with the most recent frame time
+    void updateWithNewFrame(double sample);
+    //Get the value of the most recent average frame time (seconds)
+    double getAvgFrameTime();
+
+private:
+    static const int totalSamples = 10;
```

Comment:
Coding style:  for `static const` variables prefer capital first letter (`TotalSamples`) to make it obvious that it is a constant